tutorials/simple_time_series_example.ipynb

```diff
@@ -0,0 +1,406 @@
+    "                 ('res_ext_grid', 'mdot_kg_per_s')]\n",
+    "ow = OutputWriter(net, time_steps, output_path=None, log_variables=log_variables)\n",
+    "\n",
+    "# 5. run simulation\n",
+    "run_timeseries_ppipe(net, time_steps, output_writer=ow)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The results of the simulation can be accessed via the OutputWriter as follows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ow.np_results[\"res_junction.p_bar\"])\n",
+    "ow.np_results[\"res_pipe.v_mean_m_per_s\"])\n",
+    "ow.np_results[\"res_pipe.reynolds\"]\n",
+    "ow.np_results[\"res_pipe.lambda\"]\n",
+    "ow.np_results[\"res_sink.mdot_kg_per_s\"]\n",
+    "ow.np_results[\"res_source.mdot_kg_per_s\"]\n",
+    "ow.np_results[\"res_ext_grid.mdot_kg_per_s\"]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "To view the results more clearly, they can be displayed with `matplotlib.pyplot` in a diagram with the time steps as x-axis.The time curve for the mass flow in the External Grid looks as follows:"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<img src=\"pics/time_series_simple_example_plot_mass_flow_ext_grid.png\">"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The figure can be created by the following lines:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "x = time_steps\n",
+    "y = ow.np_results[\"res_ext_grid.mdot_kg_per_s\"]\n",
+    "\n",
+    "plt.xlabel(\"time step\")\n",
+    "plt.ylabel(\"mass flow [kg/s]\")\n",
+    "plt.title(\"External Grid Mass Flow Profile\")\n",
+    "plt.plot(x, y, \"r-o\")\n",
+    "plt.grid()\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In addition, the pressure curves for the individual junctions can be plotted with this code block:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "x = time_steps\n",
+    "y = ow.np_results[\"res_junction.p_bar\"]\n",
+    "\n",
+    "plt.xlabel(\"time step\")\n",
+    "plt.ylabel(\"pressure [bar]\")\n",
+    "plt.title(\"Pressure Profile\")\n",
+    "plt.plot(x, y[:,0], \"g-o\")\n",
+    "plt.plot(x, y[:,1], \"r-o\")\n",
+    "plt.plot(x, y[:,2], \"y-o\")\n",
+    "plt.legend([\"Source\", \"External Grid\", \"Sink\"], loc='upper left')\n",
+    "plt.grid()\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<img src=\"pics/time_series_simple_example_plot_pressures.png\">"
```

Comment:
The pressure curves look a bit odd to me. How can ext_grid have the lowest pressure in the net (for t < 8), while it still delivers mass to the sink junction?

Reply:
Ok, now I get it... The sign is consumption oriented, so it is actually consuming the surplus gas from the source